blead/patches/006.patch

```diff
@@ -162,81 +162,6 @@ index 0000000..5b16031
 +    if (_CVTSTATE_OFF == orig_cvstate)
 +        __ae_autoconvert_state(_CVTSTATE_ON);
 +
-+    // Sets the LIBPATH relative to the perl executable
-+    char path[_XOPEN_PATH_MAX];
-+    char parent[_XOPEN_PATH_MAX];
-+
-+    W_PSPROC buf;
-+    int token = 0;
-+    pid_t mypid = getpid();
-+    memset(&buf, 0, sizeof(buf));
-+    buf.ps_pathlen = _XOPEN_PATH_MAX;
-+    buf.ps_pathptr = path;
-+
-+    while ((token = w_getpsent(token, &buf, sizeof(buf))) > 0) {
-+      // Found our process.
-+      if (buf.ps_pid == mypid) {
```

Comment:
style suggestion - check for != mypid and continue and then you don't need a really long piece of code in the conditional

Reply:
Also added